### ISSUE TITLE
Set the default branch to `src`

### ISSUE THREAD

**Issue by ExE-Boss** [viperHTML/viperhtml.github.io#25]
1. Go to [the **Branches** tab of the the repository **Settings**](https://github.com/viperHTML/viperhtml.github.io/settings/branches).
2. Click on the **Default branch** dropdown.
3. Select the [**`src`**](https://github.com/viperHTML/viperhtml.github.io/tree/src) branch.
4. Click the newly displayed **Update** button.
5. Profit!

---

Also, this won’t affect the organisation website build branch, which will still remain the `master` branch, but this will prevent PRs like #24 being opened on the wrong branch.

**Comment by WebReflection** [comment]
Done

**Comment by WebReflection** [comment]
So ... `travis-push-update` was pointing at `../gh-pages` but there's not such branch in here so now it's pointing at `../master` ... is this correct?

TL;DR Travis has access but it's not updating anything when I change the layout
